README.md

```diff
@@ -1,4 +1,55 @@
+![puta](https://github.com/PactCoffee/js-ui-challenge/raw/master/puta.png)
+
 Pact's front end challenge
 ==========================
 
-TBA
+The general idea
+----------------
+
+We have a sign up flow affectionately known as "The Funnel". It's primary use is to sign up customers to our service. They enter their information, it gets sent to the server. **The challenge is to build a very simple version of [our current funnel](http://pactcoffee.com/funnel/intro)**. Sounds easy enough, right?
+
+
+But wait! There's more...
+-------------------------
+
+1. The funnel is separated into "stages". Each stage collects certain information from the user. For this challenge, those stages are:
```

Comment:
Thoughts on [this](https://github.com/iest/js-ui-challenge/commit/a84152a384a62e96ec47b8169705a8dbcb269214)?